Tasks/UniversalPackagesV0/package.json

```diff
@@ -17,11 +17,8 @@
     },
     "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
     "dependencies": {
-        "@types/mocha": "^5.2.5",
+        "azure-pipelines-task-lib": "2.8.0",
         "packaging-common": "file:../../_build/Tasks/Common/packaging-common-1.0.1.tgz",
-        "utility-common": "file:../../_build/Tasks/Common/utility-common-1.0.2.tgz",
-        "vso-node-api": "6.5.0",
-        "vsts-task-lib": "2.6.0",
-        "vsts-task-tool-lib": "0.4.1"
```

Comment:
Double checking and making sure that this is intentional

Reply:
Yes, the removed libraries were all no longer used.